terraform/environment/region/modules/lambda/variables.tf

```diff
@@ -59,3 +59,8 @@ variable "ecr_arn" {
 variable "kms_key" {
   description = "KMS key for the lambda log group"
 }
+
+variable "iam_policy_document" {
+  type    = string
```

Comment:
if we make this a list of strings type, with a default value of an empty list, we can utilise the merging feature inside the module. As it stands we can only pass in one policy document json.